frontend/src/lib/components/CommandBar/SearchResults.tsx

```diff
@@ -0,0 +1,60 @@
+import { useActions, useValues } from 'kea'
+import { useEventListener } from 'lib/hooks/useEventListener'
+import { DetectiveHog } from '../hedgehogs'
+
+import { searchBarLogic } from './searchBarLogic'
+import SearchResult, { SearchResultSkeleton } from './SearchResult'
+
+const SearchResults = (): JSX.Element => {
+    const { filterSearchResults, searchResponseLoading, activeResultIndex, keyboardResultIndex, maxIndex } =
+        useValues(searchBarLogic)
+    const { onArrowUp, onArrowDown, openResult } = useActions(searchBarLogic)
+
+    useEventListener('keydown', (event) => {
```

Comment:
Sometimes I see event listeners in the logic which would really simplify this component

```
events(({ props, actions, cache }) => ({
  afterMount: () => {
      cache.onKeyDown = (e: KeyboardEvent) => {
          if (!filterSearchResults) {
              return
          }

          if (event.key === 'Enter') {
              event.preventDefault()
              openResult(activeResultIndex)
          } else if (event.key === 'ArrowDown') {
              event.preventDefault()
              onArrowDown(activeResultIndex, maxIndex)
          } else if (event.key === 'ArrowUp') {
              event.preventDefault()
              onArrowUp(activeResultIndex, maxIndex)
          }
      }
      window.addEventListener('keydown', cache.onKeyDown)
  },
  beforeUnmount: () => {
      window.removeEventListener('resize', cache.onKeyDown)
  },
})),
```

Reply:
That totally makes sense, but I'm going to postpone the change for the follow-up PR https://github.com/PostHog/posthog/pull/18355.

Reply:
https://github.com/PostHog/posthog/pull/18355/commits/cd0c8c4b81265111f8894e6dde94bfbc25bf653b